### ISSUE TITLE
Can we use this as a linter?

### ISSUE THREAD

**Issue by bixu** [McSick/honeycomb-derived-column-language-extension#2]
I'd love to be able to use the logic here as a linter, similar to `yamllint`.

**Comment by McSick** [comment]
So like a command line version to do checking?

**Comment by bixu** [comment]
Yup -- exiting non-zero if passed a `.honeycomb` file with a formatting error would be enough to start with. Line numbers in the output for lines with errors would be nice icing.
